### PR TITLE
minor readme update

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
minor readme update

### CODE REVIEW

**Comment by damienomurchu** [comment]
adding an `lgtm` seeing as @psturc doesn't seem to be a code-owner

/lgtm

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Currently the operator installs the following products:
 - Integreatly solution explorer
 
 ## Prerequisites
-- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v0.15.1.
+- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v0.15.2.
```
Comment (austincunningham) [inline README.md]: The moc is still pointing at 0.15.1
https://github.com/integr8ly/integreatly-operator/blob/a6df320d56ead7e810fd5202416904dd942dec93/go.mod#L25

Comment (austincunningham) [reply README.md]: still pointing at v0.15.1 in a few places https://github.com/integr8ly/integreatly-operator/search?q=v0.15.1&unscoped_q=v0.15.1

**Comment by pawelpaszki** [comment]
closed in favour of this: https://github.com/integr8ly/integreatly-operator/pull/415

**Pull request closed without merging**